docs/en/sql-reference/functions/string-functions.md

```diff
@@ -1150,3 +1150,13 @@ A text with tags .
 The content within <b>CDATA</b>
 Do Nothing for 2 Minutes 2:00 &nbsp;
 ```
+
+## ascii(s) {#ascii}
+
+Returns the ASCII code point of the first character of str.  The result type is Int32.
+
+If s is empty, the result is 0. If the first character is not an ASCII character or not part of the Latin-1 Supplement range of UTF-16, the result is undefined.
```

Comment:
Minor: It would be nice to add "not" also to the in-source docs (l. 79, ascii.cpp). (It is unfortunate, that the official docs are currently not generated from the in-source docs, so there is a bit of redundancy necessary.)